src/overlays/actors/ovl_En_And/z_en_and.c

```diff
@@ -30,22 +29,154 @@ ActorInit En_And_InitVars = {
+        }
+    } else if (this->hasAction) {
+        this->hasAction = false;
+        EnAnd_ChangeAnim(this, this->prevAnimIndex);
+    }
+}
+
+void EnAnd_Init(Actor* thisx, PlayState* play) {
+    EnAnd* this = THIS;
+
+    ActorShape_Init(&this->actor.shape, 0.0f, ActorShadow_DrawCircle, 14.0f);
+    SkelAnime_InitFlex(play, &this->skelAnime, &gAndSkel, NULL, this->jointTable, this->morphTable,
+                       OBJECT_AND_LIMB_MAX);
+    this->animIndex = -1;
+    EnAnd_ChangeAnim(this, 0);
+    Actor_SetScale(&this->actor, 0.01f);
+    this->actor.flags &= ~ACTOR_FLAG_1;
+    this->flags |= 8;
+    this->actionFunc = EnAnd_HandleCsAction;
+}
+
+void EnAnd_Destroy(Actor* thisx, PlayState* play) {
+}
+
+void EnAnd_Update(Actor* thisx, PlayState* play) {
+    EnAnd* this = THIS;
+
+    this->actionFunc(this, play);
+    SkelAnime_Update(&this->skelAnime);
+    EnAnd_Blink(this);
+}
+
+void EnAnd_TransformLimbDraw(PlayState* play, s32 limbIndex, Actor* thisx) {
+    EnAnd* this = THIS;
+    s32 stepRot;
+    s32 overrideRot;
+
+    stepRot = this->flags & 8 ? false : true;
+    overrideRot = this->flags & 2 ? true : false;
+    if (!stepRot) {
+        overrideRot = false;
+    }
+
+    if (limbIndex != OBJECT_AND_LIMB_TORSO) {
+        if (limbIndex == OBJECT_AND_LIMB_HEAD) {
+            SubS_UpdateLimb(this->headRotZ + this->torsoRotZ + 0x4000,
+                            this->headRotY + this->torsoRotY + this->actor.shape.rot.y + 0x4000, &this->headComputedPos,
+                            &this->headComputedRot, stepRot, overrideRot);
+            Matrix_Pop();
+            Matrix_Translate(this->headComputedPos.x, this->headComputedPos.y, this->headComputedPos.z, MTXMODE_NEW);
+            Matrix_Scale(this->actor.scale.x, this->actor.scale.y, this->actor.scale.z, MTXMODE_APPLY);
+            Matrix_RotateYS(this->headComputedRot.y, MTXMODE_APPLY);
+            Matrix_RotateXS(this->headComputedRot.x, MTXMODE_APPLY);
+            Matrix_RotateZS(this->headComputedRot.z, MTXMODE_APPLY);
+            Matrix_Push();
+        }
+    } else {
+        SubS_UpdateLimb(this->torsoRotZ + 0x4000, this->torsoRotY + this->actor.shape.rot.y + 0x4000,
+                        &this->torsoComputedPos, &this->torsoComputedRot, stepRot, overrideRot);
+        Matrix_Pop();
+        Matrix_Translate(this->torsoComputedPos.x, this->torsoComputedPos.y, this->torsoComputedPos.z, MTXMODE_NEW);
+        Matrix_Scale(this->actor.scale.x, this->actor.scale.y, this->actor.scale.z, MTXMODE_APPLY);
+        Matrix_RotateYS(this->torsoComputedRot.y, MTXMODE_APPLY);
+        Matrix_RotateXS(this->torsoComputedRot.x, MTXMODE_APPLY);
+        Matrix_RotateZS(this->torsoComputedRot.z, MTXMODE_APPLY);
+        Matrix_Push();
+    }
+}
+
+void EnAnd_Draw(Actor* thisx, PlayState* play) {
+    static TexturePtr sMouthTextures[] = { gAndMouthNeutralTex, gAndMouthSmileTex };
+    static TexturePtr sEyeTextures[] = {
```

Comment:
```suggestion
    static TexturePtr sEyeTextures[EYE_TEXTURES_COUNT] = {
```
Since you use a define, this ensures it's always the right size